CHANGELOG/CHANGELOG-1.7.0.md

```diff
@@ -1,3 +1,9 @@
+- [v1.7.0](#v170)
+  - [Changelog since v1.6.1](#changelog-since-v161)
+  - [Changes by Kind](#changes-by-kind)
+    - [Features](#features)
+    - [Bugs](#bugs)
+
 # v1.7.0 
```

Comment:
Is this should be in first line or may be deleted?

Reply:
Thanks for the feedback @alikdell.  This is the structure we've been using for the other changelog files.